src/service-worker/core.js

```diff
@@ -107,36 +97,15 @@ const cachePromise = caches.open('cdn-js').then(result => {
 });
 
 /**
- * A promise to open up our Indexed DB database, which will be resolved before
- * any requests are intercepted by the SW.
- *
- * @type {!Promise}
- */
-const dbPromise = Promise.resolve(indexedDBP.open('cdn-js', 1, {
-  upgrade(db, event) {
-    const oldVersion = event.oldVersion;
-    // Do we need to create our database?
-    if (oldVersion == 0) {
-      const files = db.createObjectStore('js-files', {autoIncrement: true});
-      files.createIndex('files', 'file');
-      files.createIndex('fileVersions', ['file', 'version']);
-    }
-  },
-}).then(result => {
-  db = result;
-}));
-
-/**
  * Fetches the request, and stores it in the cache. Since we only store one
  * version of each file, we'll prune all older versions after we cache this.
  *
  * @param {!Request} request
+ * @param {string} requestFile the basename of the request
+ * @param {string} requestVersion the version of the request
+ * @return {!Promise<Response>}
```

Comment:
nit: I believe `fetch()` is guaranteed to have a non-null responsive in the resolve case.